main/inc/lib/tracking.lib.php

```diff
@@ -388,8 +388,7 @@ public static function getLpStats(
                         $sql = "SELECT results_disabled
                                 FROM $TBL_QUIZ
                                 WHERE
-                                    c_id = $course_id AND
-                                    id ='".$my_path."'";
+                                    iid ='".$my_path."'";
```

Comment:
Quotes not necessary (but the `$my_path` variable could benefit from a renaming to give it the right meaning)